sdk/storage/Azure.Storage.Blobs/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 12.6.0-preview.1 (Unreleased)
+## 12.6.0 (20202-08-31)
```

Comment:
should this be 12.5.2?

Reply:
We have an interface change (grouped `BlobBaseClient.OpenRead()` optional parameters into `BlobOpenReadOptions`), so we have a minor version bump.

Reply:
There's an extra 2 after 2020

Reply:
Fixed, thanks!